<?xml version="1.0"?>
<launch>
  	<arg name="cloud_in" default="cylinders_pcl"/>
  	<arg name="rviz_file" default="tests.rviz"/>
	<arg name="config_file" default="config.yaml"/>

	<node pkg="shape_detection_fitting" type="planar_top_test" name="planar_top_test" args="$(find shape_detection_fitting)/config/$(arg config_file)" output="screen">
		<remap from="cloud_in" to="$(arg cloud_in)"/>
	</node>

	<node pkg="rviz" type="rviz" name="rviz" args="-d $(find shape_detection_fitting)/rviz/$(arg rviz_file)"/>
</launch>
